### PR TITLE
ADD hyperlinks and node colors.

### PR DESCRIPTION
**Pull request by ntavares**
Adding hyperlinks for SVG (google charts seems to ignore them just fine) and node colors. Colors are determined by status category (white=To Do, yellow=Doing, green=Done). Another option would be to translate from status[colorName], but I noticed SVG colors don't recognize JIRA ones, so that'd be extra work without certainty of addressing them all (for now).

Unfortunately, our workflows are a bit more complex than the 3 basic states, and I'd like to have different colors for it, so I ended up doing a different customized function for us, using status[name] instead of status[statusCategory][name].

### CODE REVIEW

**Comment by pawelrychlik** [comment]
Nice, this is actually very handy :) Thanks!

File: jira-dependency-graph.py [hunk 1]
```diff
@@ -65,14 +70,24 @@ def build_graph_data(start_issue_key, jira, excludes, show_directions, direction
...
         # truncate long labels with "...", but only if the three dots are
         # replacing more than two characters -- otherwise the truncated
         # label would be taking more space than the original.
         if len(summary) > MAX_SUMMARY_LENGTH+2:
             summary = summary[:MAX_SUMMARY_LENGTH] + '...'
         summary = summary.replace('"', '\\"')
-        return '"{}({})"'.format(issue_key, summary.encode('utf-8'))
+        log('node ' + issue_key + ' status = ' + str(status))
+        if islink:
+            return '"{}\\n({})"'.format(issue_key, summary.encode('utf-8'))
```
Comment (pawelrychlik) [inline jira-dependency-graph.py]: @ntavares why don't we add `href` and the other params for `islink` too? Can you give an example?

Comment (ntavares) [reply jira-dependency-graph.py]: Hi @pawelrychlik, links (as in arrows) don't really map to an entity in JIRA right? As for the rest, they are "fill" properties, which also don't apply to nodes (I actually didn't verify if the attributes are supported, but you're already managing color elsewhere in the code). Does that make sense? Do you have an use case where have an hyperlink on a link (arrow) would be usefull?

Comment (pawelrychlik) [reply jira-dependency-graph.py]: Ok, it makes sense, thanks.

**Pull request merged**